src/components/tree-views-container.js

```diff
@@ -161,9 +161,11 @@ const TreeViewsContainer = () => {
         (children) => {
             dispatch(
                 setCurrentChildren(
-                    children.filter(
-                        (child) => child.type !== elementType.DIRECTORY
-                    )
+                    children
```

Comment:
Not sure if it's the right place to sort... I think it's better to do it during rendering like in renderTree for TreeItems...
Maybe move this sorting code somewhere in directory-content render method ?